src/api/stats.ts

```diff
@@ -269,6 +274,45 @@ const getStatsForDetails = (
         .returns<CatalogStats_Details[]>();
 };
 
+export interface DefaultStatsWithDocument extends DefaultStats {
+    flow_document: any;
+}
+
+const getStatsForDashboard = (
+    tenant: string,
+    grain: Grains,
+    endDate: DateTime,
+    duration?: Duration
+    // entityType?: Entity
+) => {
+    const past = duration ? endDate.minus(duration) : endDate;
+
+    // let query: string;
+    // switch (entityType) {
+    //     case 'capture':
+    //         query = CAPTURE_QUERY;
+    //         break;
+    //     case 'materialization':
+    //         query = MATERIALIZATION_QUERY;
+    //         break;
+    //     case 'collection':
+    //         query = COLLECTION_QUERY;
+    //         break;
+    //     default:
+    //         query = DEFAULT_QUERY;
+    // }
+
+    return supabaseClient
+        .from(TABLES.CATALOG_STATS)
+        .select(`${DEFAULT_QUERY},flow_document`)
```

Comment:
It looks like we only ever get taskStats... can we just have that be fetched?

`flow_document:flow_document->>taskStats::json`